.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,18 +1,2 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.6/apache-maven-3.9.6-bin.zip
-wrapperUrl=https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/3.2.0/maven-wrapper-3.2.0.jar
+distributionType=only-script
```

Comment:
we don't deploy it to maven central, so I'm hoping that this won't be a problem (as we had with others where the bin type was required as otherwise CI proeprties were not picked up correctly)

Reply:
Wrapper was generated with the older version of the plugin (3.2.0). If we update the wrapper plugin version, we probably should also re-generate the wrapper itself (otherwise, what would be the point of bumping the plugin version? 🙂)
The default distribution type has changed and now it is a `script-only` (https://maven.apache.org/tools/wrapper/#Usage_with_or_without_Binary_JAR) which is "nice" since it ensuires that there are no binaries included in the sources.

Reply:
should I move that comment ^ somewhere else ? 🙈 🫣

Reply:
added to the commit message too 🙂